client/state/reader/lists/actions.js

```diff
@@ -298,6 +300,27 @@ export const receiveReaderListItems = ( listId, listItems ) => ( {
 	listItems,
 } );
 
+export const deleteReaderListFeed = ( listOwner, listSlug, feedId ) => ( {
+	type: READER_LIST_ITEM_DELETE_FEED,
+	listOwner,
+	listSlug,
+	feedId,
+} );
+
+export const deleteReaderListSite = ( listOwner, listSlug, siteId ) => ( {
+	type: READER_LIST_ITEM_DELETE_FEED,
```

Comment:
What's the rationale for the reuse of `READER_LIST_ITEM_DELETE_FEED` rather than a new `READER_LIST_ITEM_DELETE_SITE`?

Reply:
It maps to the same api action, just with `site:` prefixed to the site id. 

But yeah it might make it easier for the reducer to break it out by key type. I'll switch it up.